xmljson/__init__.py

```diff
@@ -223,16 +223,10 @@ def data(self, root):
                     children_list = [self._fromstring(text), ]
 
         count = Counter(child.tag for child in children)
```

Comment:
HI @dagwieers, thanks for the suggestion. I have removed the count variable and updated the code.